sections/semantics-forms.include

```diff
@@ -395,50 +395,11 @@
 &lt;/form&gt;
     </pre>
 
-<h5 id="improving-the-user-experience-on-mobile-devices">Improving the user experience on mobile devices</h5>
```

Comment:
Instead of just removing this section, it would be helpful to make it consistent with the use of `pattern`.

There is nothing that prevents a user agent from customising the input UI to match the expected pattern - e.g. if there are a restricted set of acceptable input characters such as numerals and the plus sign and space, it would be good to offer a keyboard that did just that.  In any event, using validity the application can directly help users... **IFF** designed well. But what isn't well-designed on the web?

Reply:
This change request seems editorial, and we can live without it.